backend/apps/ai/common/base/chunk_command.py

```diff
@@ -77,7 +79,21 @@ def process_chunks_batch(self, entities: list[Model]) -> int:
                 self.stdout.write(f"Chunks for {entity_key} are already up to date.")
 
         if batch_chunks_to_create:
-            Chunk.bulk_save(batch_chunks_to_create)
+            context_ids = {context_id for context_id, _ in batch_chunks_to_create}
```

Comment:
Any reason to not using (again) our backend-wide approach for handling models data -- model.update_data + model.bulk.save instead?

Reply:
You already have a check in the Chunk::update_data method. Why it's not enough and why it can't be handled there?